swagger.json

```diff
@@ -4088,9 +4078,14 @@
                     "description": "ID of the vehicle that has been permanently assigned to this driver.",
                     "example": "879"
                 },
-                "driver": {
-                    "$ref": "#/definitions/ExternalDriverResponse"
-                }
+                "externalIds" : {
```

Comment:
Do other endpoints (that we are not modifying to support external ids) utilize `DriverBase`?

Reply:
Double-checked, and it looks like DriverBase is only used in either the request or response of /fleet/driver/create, GET and PATCH fleet/driver/:id, and /inactive/driver/:id.